browser/admin/src/integrator/AdminIntegratorSettings.ts

```diff
@@ -682,6 +682,16 @@ document.addEventListener('DOMContentLoaded', () => {
 		initTranslationStr();
 		(window as any).settingIframe = new SettingIframe();
 		(window as any).settingIframe.init();
+		const resizeObserver = new ResizeObserver(() => {
+			window.parent.postMessage(
+				{
+					type: 'SETTING_IFRAME_HEIGHT',
```

Comment:
```suggestion
					type: 'Iframe_Height',
```
To stay consisent with existing signals..
https://sdk.collaboraonline.com/docs/postmessage_api.html#id4